src/app/Topology/Actions/NodeActions.tsx

```diff
@@ -139,11 +141,74 @@ export const nodeActions: NodeAction[] = [
   },
   { isSeparator: true },
   {
-    action: (element, { services }, track) => {
+    action: (element, { services }) => {
       const targetNode: TargetNode = element.getData();
-      track(services.api.deleteTarget(targetNode.target).subscribe(() => undefined));
+      services.api.deleteTarget(targetNode.target).subscribe(() => undefined);
     },
     title: 'Delete Target',
     includeList: [NodeType.CUSTOM_TARGET],
   },
+  {
+    action: (element, { services }) => {
+      const group: EnvironmentNode = element.getData();
+      const recordingName = `${group.name.replace(/[\s+-]/g, '_')}_${Date.now()}`;
```

Comment:
Using the timestamp as part of the name is nice to help the user avoid name collisions but it also makes it harder/costlier for us to undo that operation, since now we need to query for recordings that have names that look like they match this pattern. I think using a fixed constant for the name is okay so long as the name is unlikely to be something the user has already used on their own, ex. `cryostat_topology_action`. `cryostat_group_${group.name.replaced}` might be okay as well but I wonder if there are cases or platforms where the group name may change without that also meaning that all the children to the leaf nodes have been redeployed, in which case a group's rename would also break our ability to undo group operations.